### PR TITLE
add an integration of nodeos for crash when the nodes are killed

### PR DESCRIPTION
**Pull request by vzqzhang**
## Change Description

Added a new integration test for nodeos. This test sets up one producing nodes and one "bridge" node using test_control_api_plugin. We send transactions to the bridge node. After the accounts are initialized and allocated with symbols, we kill the producing node. Then we flood hundreds transactions to the bridge node. At the end, we ill the bridge. It tests that no crashes happen when the nodes are killed. 

## Change Type
**Select ONE**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: tests/nodeos_contrl_c_test.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    cluster.setWalletMgr(walletMgr)
+    cluster.killall(allInstances=killAll)
+    cluster.cleanup()
+
+    specificExtraNodeosArgs = {}
+    specificExtraNodeosArgs[0] = "--plugin eosio::producer_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin "
+    "--plugin eosio::txn_test_gen_plugin --plugin eosio::producer_api_plugin --plugin eosio::history_plugin "
+    "--plugin eosio::history_api_plugin --plugin eosio::state_history_plugin --plugin eosio::db_size_api_plugin "
+    "--p2p-listen-endpoint=0.0.0.0:9876 --p2p-server-address=0.0.0.0:9876 --http-server-address=127.0.0.1:8888 "
+    "--delete-all-blocks --disable-replay-opts --delete-state-history --access-control-allow-origin='*'"
```
Comment (heifner) [inline tests/nodeos_contrl_c_test.py:50]: You don't need history_plugin, history_api_plugin, state_history_plugin, db_size_api_plugin.

I'm not sure you even need anything here for extra args. @brianjohnson5972

Comment (vzqzhang) [reply tests/nodeos_contrl_c_test.py]: > You don't need history_plugin, history_api_plugin, state_history_plugin, db_size_api_plugin.
> 
> I'm not sure you even need anything here for extra args. @brianjohnson5972

Removed the extra options

**Comment by nksanthosh** [comment]
Since this test is not consistent in results - should we close the PR?

**Comment by heifner** [comment]
I asked if we could get automation to run this test a few hundred times to see if it fails a few times. If it does fail then it would be useful to keep. If it never fails, then I would like us to modify it such that it does cause develop to fail. We have hit shutdown bugs under load a number of times and it would be good to have a test for it.

**Comment by nksanthosh** [comment]
@kj4ezj Will you be able to run this test a few hundred times and report back on results? (not urgent - can be scheduled per your weekly workload)

**Comment by kj4ezj** [comment]
> @kj4ezj Will you be able to run this test a few hundred times and report back on results?

This required code changes to the CI system, so I opened [pull request 9551](https://github.com/EOSIO/eos/pull/9551) based on this branch. I successfully completed a short test run in [build 24759](https://buildkite.com/EOSIO/eosio/builds/24759) with 21 runs in rounds of 14 runs each.

Finally, I kicked off [build 24760](https://buildkite.com/EOSIO/eosio/builds/24760) to perform 2100 runs across 30 rounds of 70 runs each (ten runs per operating system).

**Review by heifner — APPROVED** [review]

**Pull request merged**